apis/config/v1alpha1/configuration_types.go

```diff
@@ -38,6 +38,11 @@ type Configuration struct {
 	// Defaults to false; therefore, those jobs are not managed and if they are created
 	// unsuspended, they will start immediately.
 	ManageJobsWithoutQueueName bool `json:"manageJobsWithoutQueueName"`
+
+	// DisableInternalCertManagement controls whether to disable internal cert management or not,
+	// default to false. If you want to use a third-party management, e.g. cert-manager,
+	// set it to true. See the user guide for more information.
+	DisableInternalCertManagement bool `json:"disableInternalCertManagement"`
```

Comment:
let's call it `EnableInternalCertManagement` and make it a pointer so that the default can be true.

Reply:
Done :)